### PR TITLE
Add IAS bot detection

### PR DESCRIPTION
**Pull request by dlackty**
Hi @alaz,

I hope you're doing well! It's been a while since my last contribution to this project. I've made some changes that I would like you to review:

1. Added IAS bots detection.
2. Updated Rubocop custom cop to support IP lists in JSON.

Please take a look and provide your feedback. Thank you!

### CODE REVIEW

File: lib/legitbot/ias.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+module Legitbot # :nodoc:
+  # https://integralads.com/ias-privacy-data-management/policies/site-indexing-policy/
+  class Ias < BotMatch
+    # @fetch:url https://integralads.com/policy-docs/iasbot.json
+    # @fetch:jsonpath $.publicIPs[*].ipv4
+    ip_ranges %w[
+      3.217.168.199
+      18.214.43.70
+      3.226.51.67
```
Comment (alaz) [inline lib/legitbot/ias.rb:11]: Could you please sort the list? Then the next automatic diff will be cleaner.

You don't have to do it manually. You can just remove a line, run `rubocop -A` and the custom rule will take care of soring the list properly.

**Review by alaz — COMMENTED** [review]
Looks good to me. I have just one minor comment. Thank you for working on this!

File: lib/legitbot/ias.rb (hunk 1 above)
Comment (dlackty) [reply lib/legitbot/ias.rb:10]: Done! Just amended the commit and forced push the branch.

**Review by alaz — APPROVED** [review]
Great. Thank you!

**Comment by alaz** [comment]
Released as https://github.com/alaz/legitbot/releases/tag/v1.8.0

**Pull request merged**